frontend/src/scenes/insights/ActionFilter/RenameModal.tsx

```diff
@@ -42,8 +42,8 @@ export function RenameModal({ typeKey, view }: RenameModalProps): JSX.Element {
             }
             onCancel={hideModal}
         >
-            Query steps can be renamed to provide a more meaningful label for your insight. Renamed steps are also shown
-            on dashboards.
+            Query series/steps can be renamed to provide a more <b>meaningful label for you and your team members</b>.
```

Comment:
Shouldn't we prefer `strong` tags to `b` tags by default?

Reply:
Done! Curious why this is the case